### ISSUE TITLE
Explicit comment to un-assign second channel

### ISSUE THREAD

**Issue by rusefillc** [rusefi/rusefi#5110]
When second channel is not needed and not selected it's not visible - everything is easy

We have to enable second channel when it's not needed but selected in order to allow user to un-select it

Well, technically should we simply programmatically un-select it? Can we?
Well, technically we can initialize physical pin based on trigger wheel shape but we do not do that _yet_.

As a shortcut, while we are looking for ways to programmatically do better at least we can tell user to REMOVE value.

![image](https://user-images.githubusercontent.com/48498823/220175117-780c6c45-4517-47df-8580-7d0437d9c69c.png)

**Comment by rusefillc** [comment]
![image](https://user-images.githubusercontent.com/48498823/220177461-89e20664-a40b-4090-9968-e25381fd4cc7.png)

**Comment by mck1117** [comment]
we know whether the trigger is going to use that input or not, why can't we just ignore it in software if we don't need the second channel?

**Comment by rusefillc** [comment]
> Well, technically we can initialize physical pin based on trigger wheel shape but we do not do that _yet_.

Yes we can. I suggest separate ticket for that improved solution.
